### PR TITLE
Add a note about a training day for first-time speakers

### PR DESCRIPTION
**Pull request by jean**
Also add a form to register interest.
Currently it's commented out: maybe we should just reach out to speakers
directly?

### CODE REVIEW

**Comment by jean** [comment]
@tonytonybell the part to translate is here: https://github.com/PyConTH/www/pull/74/files#diff-48f05d343e4189c833614b1ad2eb9316R54

**Comment by jean** [comment]
Thank you @tonytonybell ! 
One of @vtno @djay @citizen428 @frawau please review & merge?

File: site/pages/submit-talk.th.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 ขอเชิญทุกคนเข้าร่วมงาน Python Conference Thailand
 
-โดยงานจะจัดขึ้นในวันที่ 16-17 มิถุนายน 2561 ซึ่งเป็นครั้งแรกของงาน PyCon ในประเทศไทยและพวกเราต้องการความช่วยเหลือจากคุณ งาน PyCon จัดขึ้นทั่วโลกโดยโปรแกรมเมอร์ที่เขียน Python เพื่อโปรแกรมเมอร์ที่เขียน Python งานจะจัดขึ้นในวันหยุดสุดสัปดาห์เป็นเวลา 2 วัน โดยจะมีการพูดคุยทั้งภาษาไทยและภาษาอังกฤษ เราต้องการ proposal ของคุณที่จะเป็นหัวข้อของ talk lightning talk หรือการฝึกอบรม (workshop) ในงานสัมนาครั้งนี้
+โดยงานจะจัดขึ้นในวันที่ 16-17 มิถุนายน 2561 ซึ่งเป็นครั้งแรกของงาน PyCon ในประเทศไทยและพวกเราต้องการความช่วยเหลือจากคุณ งาน PyCon จัดขึ้นทั่วโลกโดยโปรแกรมเมอร์ที่เขียน Python เพื่อโปรแกรมเมอร์ที่เขียน Python งานจะจัดขึ้นในวันหยุดสุดสัปดาห์เป็นเวลา 2 วัน โดยจะมีการพูดคุยทั้งภาษาไทยและภาษาอังกฤษ เราต้องการ proposal ของคุณที่จะเป็นหัวข้อของ talks, lightning talk หรือการฝึกอบรม (workshop) ในงานสัมนาครั้งนี้
```
Comment (vtno) [inline site/pages/submit-talk.th.rst:13]: talks -> talk

File: site/pages/submit-talk.th.rst [hunk 2]
```diff
@@ -33,7 +33,7 @@
 .. raw:: html
 
 
-   <a class="btn btn-primary btn-lg active" href="https://www.papercall.io/pyconth" role="button">ส่งข้อเสนอการพูดของคุณตอนนี้!</a>
+   <a class="btn btn-primary btn-lg active" href="https://www.papercall.io/pyconth" role="button">ส่ง Proposal การพูดของคุณตอนนี้!</a>
```
Comment (vtno) [inline site/pages/submit-talk.th.rst]: ส่ง Proposal สำหรับ talk ของคุณในตอนนี้

**Comment by vtno** [comment]
@jean Made some comments. Please fix those and it's good to go. 😄

**Comment by jean** [comment]
OK done!

**Pull request merged**